### PR TITLE
Added ZenFiles to README

### PR DESCRIPTION
**Pull request by htahir1**
## Describe changes
I added ZenFiles to our main README for more visibility. Let me know if you think it makes sense!

## Pre-requisites
Please ensure you have done the following:
- [x] I have read the **CONTRIBUTING.md** document.
- [x] If my change requires a change to docs, I have updated the documentation accordingly.
- [x] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/features/integrations) table.
- [x] I have added tests to cover my changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Other (add details above)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ ZenML pipelines are designed to be written early on the development lifecycle. D
 | 🗳 **[Vote for Features]** | Pick what we work on next! |
 | 📓 **[Docs]** | Full documentation for creating your own ZenML pipelines. |
 | 📒 **[API Reference]** | The detailed reference for ZenML's API. |
+| 🗂️️ **[ZenFiles]** | End-to-end fully fledge projects using ZenML. |
```
Comment (strickvl) [inline README.md:59]:
```suggestion
| 🗂️️ **[ZenFiles]** | End-to-end projects using ZenML. |
```

**Review by ayush714 — COMMENTED** [review]
Looks good to me :)

File: README.md [hunk 2]
```diff
@@ -247,6 +249,13 @@ pipeline = mnist_pipeline(
 pipeline.run()
 ```
 
+# 🗂️ ZenFiles
+
+ZenFiles are production-grade ML use-cases powered by ZenML. They are fully fleshed out, end-to-end, projects that show-case
```
Comment (strickvl) [inline README.md:254]:
```suggestion
ZenFiles are production-grade ML use-cases powered by ZenML. They are fully fleshed out, end-to-end, projects that showcase 
```

File: README.md [hunk 3]
```diff
@@ -247,6 +249,13 @@ pipeline = mnist_pipeline(
 pipeline.run()
 ```
 
+# 🗂️ ZenFiles
+
+ZenFiles are production-grade ML use-cases powered by ZenML. They are fully fleshed out, end-to-end, projects that show-case 
+ZenML's capabilities. They also serve as a template to start similar projects from.
```
Comment (strickvl) [inline README.md:255]:
```suggestion
ZenML's capabilities. They can also serve as a template from which to start similar projects.
```

**Review by strickvl — CHANGES_REQUESTED** [review]
Looks good!

**Review by strickvl — APPROVED** [review]

**Pull request merged**